docs/modules/ROOT/pages/config-safe.adoc

```diff
@@ -100,20 +86,23 @@ Provide the encryption key to see the result:
     OK Returned 1 row(s).
 ----
 
+== Persisting the Encryption Key
+
+CLC asks for the encryption key each time you execute a `--safe` command in non-interactive mode. To avoid this, you can use the CLC in interactive mode or scripting mode.
```

Comment:
"Persisting the Encryption Key" sounds a bit like CLC is writing it on disk.
Would it make sense to have something along the lines of "CLC remembers the encryption key in a session?"